packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2475,12 +2478,25 @@ module.exports = function (log, config, oauthdb) {
 
   Mailer.prototype._generateLinks = function (
     primaryLink,
-    { email, uid },
+    message,
     query,
     templateName,
     appStoreLink,
     playStoreLink
   ) {
+    const { email, uid } = message;
+
+    const translator = this.translator(message.acceptLanguage);
+    const {
+      termsOfServiceURL = this.subscriptionTermsUrl,
+      privacyNoticeURL = this.privacyUrl,
+    } = productDetailsFromPlan(
+      {
+        product_metadata: message.productMetadata,
+      },
+      translator.language
+    );
```

Comment:
This is kind of the main event: We can get localized terms & privacy URL to use in subscription email templates here.